src/main.js

```diff
@@ -129,6 +129,12 @@ function ping(event) {
 }
 addEventListener("pagehide", ping);
 addEventListener("visibilitychange", ping);
+addEventListener("DOMContentLoaded", function() {
+  webVitals.getCLS(sendToGoogleAnalytics);
```

Comment:
This could happen after `DOMContentLoaded` and then not fire. You can remove the event listener because the script element has `defer`